src/plugins/drawing-tool/objects/drawing-object.ts

```diff
@@ -1,4 +1,5 @@
 import { getMembers, Instance, SnapshotIn, types } from "mobx-state-tree";
+import React from "react";
```

Comment:
Is this import required? It looks like you're just adding a comment to the file

Reply:
Good point. It isn't required.